src/LiveDevelopment/MultiBrowserImpl/transports/remote/PostMessageTransportRemote.js

```diff
@@ -0,0 +1,61 @@
+// This is a transport injected into the browser via a script that handles the low
+// level communication between the live development protocol handlers on both sides.
+// This transport provides a postMessage mechanism. It's injected separately from the
+// protocol handler so that the transport can be changed separately.
+
+(function (global) {
+    "use strict";
+    
+   var postMessageTransport = {
+                /**
+                 * @private
+                 * An object that contains callbacks to handle various transport events. See `setCallbacks()`.
+                 * @type {?{connect: ?function, message: ?function(string), close: ?function}}
+                 */
+                _callbacks: null,
+                
+                /**
+                 * Sets the callbacks that should be called when various transport events occur. All callbacks
+                 * are optional, but you should at least implement "message" or nothing interesting will happen :)
+                 * @param {?{connect: ?function, message: ?function(string), close: ?function}} callbacks
+                 *      The callbacks to set.
+                 *      connect - called when a connection is established to Brackets
+                 *      message(msgStr) - called with a string message sent from Brackets
+                 *      close - called when Brackets closes the connection
+                 */
+                setCallbacks: function (callbacks) {
+                    if (!global._Brackets_LiveDev_Socket_Transport_URL) {
```

Comment:
What's happening here? This looks like websocket code that survived the culling.